testpar/t_subfiling_vfd.c

```diff
@@ -2347,7 +2347,8 @@ main(int argc, char **argv)
         must_unset_ioc_thread_count_env = true;
     }
 
-    if (!(env_value = HDgetenv(H5FD_SUBFILING_CONFIG_FILE_PREFIX))) {
```

Comment:
If env_value is not used anywhere else just remove "env_value="  part?

Reply:
Yes, that would be the correct fix here.